tests/test_qaoa.py

```diff
@@ -660,10 +671,29 @@ def circuit(params, **kwargs):
             qml.layer(qaoa_layer, 2, params[0], params[1])
 
         # Defines the device and the QAOA cost function
-        dev = qml.device('default.qubit', wires=len(wires))
+        dev = qml.device("default.qubit", wires=len(wires))
         cost_function = qml.ExpvalCost(circuit, cost_h, dev)
 
         res = cost_function([[1, 1], [1, 1]])
         expected = -1.8260274380964299
 
         assert np.allclose(res, expected, atol=tol, rtol=0)
+
+
+class TestCycles:
+
+    """Test `cycles` functions are behaving correctly"""
+
+    def test_edges_to_wires(self):
+        """Test that map_edges_to_wires returns the correct mapping"""
+        g = nx.lollipop_graph(4, 1)
+        r = edges_to_wires(g)
+
+        assert r == {(0, 1): 0, (0, 2): 1, (0, 3): 2, (1, 2): 3, (1, 3): 4, (2, 3): 5, (3, 4): 6}
```

Comment:
Just realized that these tests don't cover a directed graph :thinking: Could we copy/paste to get two more similar tests, but for a directed graph? e.g., by adding `.to_directed()` when constructing the graph.

Reply:
Can do!

Reply:
addressed here: 7cae0c9adf04197d0daba37c410d3f52c996243c